### ISSUE TITLE
Visualization: choose a framework

### ISSUE THREAD

**Issue by nicola** [densekernel/3001#10]
- angularjs ?
- jquery ?

**Comment by himerzi** [comment]
I see this as a subset of #11. We'll let this issue follow from the work done for #11.

**Comment by nicola** [comment]
Yup (as in the other one, I generally tend to create more issue for littler choices), feel free to open and close issues
